src/platform/user/profile/vet360/components/PhoneField/PhoneEditModal.jsx

```diff
@@ -72,6 +76,7 @@ class PhoneEditModal extends React.Component {
         inputPhoneNumber:
           this.props.data &&
           [this.props.data.areaCode, this.props.data.phoneNumber].join(''),
+        'view:showSMSCheckbox': this.props.showSMSCheckbox,
```

Comment:
This `view:showSMSCheckbox` data won't actually be used as "view" data at all, but instead used by the form in a `hideIf()` to determine if the checkbox should be shown at all.